eng/pipelines/templates/jobs/ci.yml

```diff
@@ -423,21 +423,29 @@ jobs:
               - '!sdk/**/test-recordings'
               - '!sdk/**/session-records'
 
-      - template: /eng/pipelines/templates/steps/generate-project-list-and-cache-maven-repository.yml
-        parameters:
-          Artifacts: ${{ parameters.Artifacts }}
-          AdditionalModules: ${{ parameters.AdditionalModules }}
-          JobType: 'Compliance'
-          UseCache: false
+      # For PullRequests CredScan will be run against the files changed in the PR.
```

Comment:
CredScan should already be filtering by changed files in a PR see https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/pipelines/templates/steps/credscan.yml#L14. Is that not what you are seeing?

Reply:
I don't have the context of what Alan is doing here, but I'll try to fill in some of the blanks as he is finished for the day. Alan did this change based on my [PR here](https://github.com/Azure/azure-sdk-for-java/pull/35810), which was seeing [credscan failures](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2910461&view=logs&j=bc67675d-56bf-581f-e0a2-208848ba68ca&t=27a9db8a-8f0c-5ed5-0322-7c4d0e71b2e0&l=94).

Reply:
Thanks for the context.